### PR TITLE
AB#39095 Added documentation link buttons

### PR DESCRIPTION
**Pull request by jerryxihe**
ADO Board Link: [Task #39095](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_workitems/edit/39095)

## Overview
* We needed buttons linked to Graph Explorer and documentation.

### Demo

https://user-images.githubusercontent.com/46834951/126411526-07b5e869-80c9-419f-bb3e-56c02d965fcf.mp4

### Notes
* Wait for https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/18 to be merged before merging this PR to `interns/dev`

## Testing Instructions
* Pull and run the Teams app.

### CODE REVIEW

File: ToolkitApp/tabs/src/components/sample/DocumentationLinks.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { GRAPH_EXPLORER_URL, GRAPH_EXPLORER_DOCS_URL, OFFICIAL_RSC_URL} from "./TabConstants";
+import { Button, Flex } from '@fluentui/react-northstar';
+
+const DocumentationLinks = () => {
+    return (
+        <Flex gap="gap.large">
+            <Button content="Go to Graph Explorer" onClick={() => window.open(GRAPH_EXPLORER_URL)} primary />
+            <Button content="Graph Explorer Documentation" onClick={() => window.open(GRAPH_EXPLORER_DOCS_URL)} primary />
+            <Button content="Resource-Specific Consent" onClick={() => window.open(OFFICIAL_RSC_URL)} primary />
```
Comment (lhao03) [inline ToolkitApp/tabs/src/components/sample/DocumentationLinks.jsx]: we should put these in a constants file later

**Review by lhao03 — APPROVED** [review]
NOICE 
DodoskyTimmyGIF

**Review by ayubun — APPROVED** [review]
UI and UX-wise, this looks absolutely wonderful! I'll test later if you'd like some extra testing-hands, but I also don't think you need my testing if both you and Lucy verified it ^^

**Review by gracet9182 — APPROVED** [review]
╰(*°▽°*)╯nice

File: ToolkitApp/tabs/src/components/sample/DocumentationLinks.jsx (hunk 1 above)
Comment (jerryxihe) [reply ToolkitApp/tabs/src/components/sample/DocumentationLinks.jsx]: 905385ceec5df828525657bc254d1c827022311a Added localization support, which involves moving these strings to a constants file. Tested it in English and "Chinese".

**Pull request merged**